lib/services/build.ts

```diff
@@ -125,7 +126,7 @@ export class BuildService implements Project.IBuildService {
 				} else if (settings.buildForTAM) {
 					this.$logger.warn("You have not specified certificate to code sign this app. We'll use default debug certificate. " +
 						"Use --certificate option to specify your own certificate. You can check available certificates with '$ appbuilder certificate' command.");
-				} else if (settings.configuration === "Release") {
+				} else if (settings.buildConfiguration === "Release" ) {
```

Comment:
@TsvetanMilanov valid point

@rosen-vladimirov 
`settings.buildConfiguration` is either `constants.Configurations.Release` or `constants.Configurations.Debug` so no lowercase is required - I can add it if you insist, however it's unnecessary